resonance/pipe/baseline.py

```diff
@@ -0,0 +1,35 @@
+import resonance.si as si
+import resonance.db as db
+from resonance.internal import declare_transformation, Processor
+import numpy as np
+
+
+@declare_transformation
+class baseline(Processor):
+
+    def prepare(self, input, beginOffset, endOffset):
+        if not isinstance(input, db.Window):
```

Comment:
please focus on the tests first, place here unconditional exception to force tests fail directly